src/components/pages/users.js

```diff
@@ -160,10 +162,57 @@ function Users({ initialState, PageTemplate }) {
     }
   };
 
+  const handleStoragePlanChange = async (user, newStoragePlanId) => {
+    const oldStorage = user.storage;
+    const newStorage = oldStorage
+      ? {
+        ...oldStorage,
+        plan: newStoragePlanId
+      }
+      : {
+        plan: newStoragePlanId,
+        usedStorageInBytes: 0,
+        reminders: []
+      };
```

Comment:
Same here, we could use a `getDefaultStorage` and reduce this to

`const newStorage = { ...(oldStorage || getDefaultStorage()), plan: newStoragePlanId }`

Reply:
Done